crypto/src/proofs/transparent.rs

```diff
@@ -809,7 +812,7 @@ mod tests {
         };
 
         assert!(proof
-            .verify(-value_to_send.commit(v_blinding), note.commit(), epk)
```

Comment:
minor: there are two other places in these transparent proof tests where you can remove the signs on value just to prevent confusion in the future (lines 851, 917)

Reply:
Thanks!